src/main/java/org/shanerx/tradeshop/listeners/ShopCreateListener.java

```diff
@@ -124,17 +123,11 @@ public void onSignChange(SignChangeEvent event) {
 		ItemStack product = lineCheck(event.getLine(1)),
 				cost = lineCheck(event.getLine(2));
 
-		if (product != null)
+        if (product != null && shop.getProduct().size() == 0)
```

Comment:
Why `==0`?

Reply:
Because its checking if the array is  empty. I think #isEmpty() may be an alternative same with <1 but they all kinda do the same thing...